requirements.txt

```diff
@@ -1,6 +1,4 @@
-python_dateutil >= 2.5.3
-setuptools >= 21.0.0
-urllib3 >= 1.25.3, < 2.1.0
-pydantic >= 1.10.5, < 2
-aenum >= 3.1.11
-websocket-client>=1.6.1
\ No newline at end of file
+pydantic~=2.4.2
```

Comment:
same as above for the requirements: openapi generator currently uses pydantic < 2 and, since there are breaking changes bumping it, I would stick with the original ones, if there are not particular reasons to bump it